tests/unit/zstackctl/test_management_ipv6_helpers.py

```
@@ -169,3 +169,49 @@ def test_add_ip6_rejects_invalid_input():
+
+def test_prepare_ipv6_system_parameters_fails_when_kernel_disables_ipv6():
+    try:
+        management_network_ipv6.prepare_ipv6_system_parameters(
+            lambda command: None,
+            proc_exists_func=lambda path: True,
+            read_file_func=lambda path: 'BOOT_IMAGE=/vmlinuz ipv6.disable=1',
+        )
+    except management_network_ipv6.IPv6SystemParameterError as e:
+        assert 'ipv6.disable=1' in str(e)
+    else:
+        assert False, 'expected IPv6SystemParameterError'
```

Ruff (0.15.14)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)